json/tbd_seb_n3.json

```diff
@@ -0,0 +1,48 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing TBD JSON for seb.osm",
+  "psis": [
+    {
+      "id": "good",
+      "parapet": 0.5,
+      "rimjoist": 0.9
+    },
+    {
+      "id": "compliant",
+      "rimjoist": 0.3,
+      "parapet": 0.325,
+      "fenestration": 0.35,
+      "concave": 0.45,
+      "convex": 0.45,
+      "balcony": 0.5,
+      "party": 0.5,
+      "grade": 0.45
+    }
+  ],
+  "khis": [
+    {
+      "id": "column",
+      "point": 0.5
+    },
+    {
+      "id": "support",
+      "point": 0.5
+    }
+  ],
+  "surfaces": [
+    {
+      "id": "Entryway  Wall 5",
+      "khis": [
+        {
+          "id": "column",
+          "count": 3
+        },
+        {
+          "id": "support",
+          "count": 4
+        }
+      ],
+      "psi": "good"
+    }
+  ]
+}
```

Comment:
Currently too many tests (initially useful for validating incremental variations while developing). Only the later versions (ex. n6, n7) will likely be kept at release.

Reply:
Where would you reference the default `psi` set for the entire model?

Reply:
If there is no default `psi` specified does that mean no derating?

Reply:
Also, I don't think you can really ever have too many tests, I would recommend keeping them all.

Reply:
The (optional) :unit PSI set (on file) is the default one for the entire model. If it is absent from the JSON file, then the TBD Measure PSI set pull-down option is instead used as the default one for the entire model. If both are set/provided, the :unit one on file supersedes the pull-down option.